hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenGetNftInfoHandler.java

```diff
@@ -123,19 +125,22 @@ public Response findResponse(@NonNull final QueryContext context, @NonNull final
     private Optional<TokenNftInfo> infoForNft(
             @NonNull final NftID nftId,
             @NonNull final ReadableNftStore readableNftStore,
+            @NonNull final ReadableTokenStore readableTokenStore,
```

Comment:
Can you also update the javadoc comment with the missing param?

Reply:
Ah yes, forgot about javadoc, thank you. Updated.